dynamiqs/utils/operators.py

```diff
@@ -330,6 +334,94 @@ def squeeze(
     return torch.matrix_exp(0.5 * (z.conj() * a2 - z * a2.mH))
 
 
+def quadrature(
+    dim: int,
+    phi: float,
+    *,
+    dtype: torch.complex64 | torch.complex128 | None = None,
+    device: str | torch.device | None = None,
+) -> Tensor:
+    r"""Returns the quadrature operator of phase angle $\phi$.
+
+    It is defined by $x(\phi) = (e^{i\phi} a + e^{-i\phi} a^\dag) / 2$, where $a$ and
+    $a^\dag$ are the annihilation and creation operators respectively.
```

Comment:
```suggestion
    It is defined by $x(\phi) = (e^{i\phi} a + e^{-i\phi} a^\dag) / 2$, where $a$ and
    $a^\dag$ are the annihilation and creation operators, respectively.
```
Extreme nit.

Reply:
I like the large formulas, I think it's easier to read and captures the eye, but it's as you prefer 😉